Package.swift

```diff
@@ -71,6 +71,12 @@ let package = Package(
             name: "AzureCommunicationTests",
             dependencies: ["AzureCommunication"],
             path: "sdk/communication/AzureCommunication",
+            exclude: [
+                "Tests/AzureCommunicationTests-Bridging-Header.h",
+                "Tests/ObjCCommunciationUserCredentialTests.m",
+                "Tests/ObjCCommunicationUserCredentialAsyncTests.m",
+                "Tests/ObjCTokenParserTests.m"
+            ],
```

Comment:
Is there a way to wildcard exclude *.m and *.h?  If not, this is okay for now.

Reply:
@tjprescott Doing a bit more research on this. But according to this post from Jun 22 2020 https://forums.swift.org/t/support-for-wildcard-paths-in-package-swift/37814 it looks like it was pitched but not able to yet.